libpng/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: "1.6.37"
```

Comment:
1.6.37 has a security vulnerability CVE-2019-7317. We should use at least 1.6.39.

Reply:
Yes， I tested the main branch， it‘s ok for me